<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Ascii Design 0.1</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res.qrc" >
    <normaloff>:/Toolbar/pics/ascii-design.png</normaloff>:/Toolbar/pics/ascii-design.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" name="horizontalLayout_3" >
    <item>
     <widget class="QToolBox" name="toolBox" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="page" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>515</width>
         <height>300</height>
        </rect>
       </property>
       <attribute name="label" >
        <string>Normal Ascii</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QTextEdit" name="textEditNormal" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>94</width>
         <height>68</height>
        </rect>
       </property>
       <attribute name="label" >
        <string>Ascii Art Text</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QTextEdit" name="textEditFiglet" />
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenFile" />
    <addaction name="separator" />
    <addaction name="actionSave" />
    <addaction name="actionSaveAs" />
    <addaction name="separator" />
    <addaction name="actionClose" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionConfigure" />
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp" />
    <addaction name="actionInfo" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menuEdit" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile" />
   <addaction name="separator" />
   <addaction name="actionSave" />
   <addaction name="actionSaveAs" />
   <addaction name="separator" />
   <addaction name="actionConfigure" />
   <addaction name="separator" />
   <addaction name="actionHelp" />
   <addaction name="actionInfo" />
   <addaction name="separator" />
  </widget>
  <action name="actionOpenFile" >
   <property name="icon" >
    <iconset resource="../res.qrc" >
     <normaloff>:/Toolbar/pics/fileopen.png</normaloff>:/Toolbar/pics/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>Open</string>
   </property>
   <property name="statusTip" >
    <string>Open a text file...</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="icon" >
    <iconset resource="../res.qrc" >
     <normaloff>:/Toolbar/pics/filesave.png</normaloff>:/Toolbar/pics/filesave.png</iconset>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
   <property name="statusTip" >
    <string>Save current ascii art...</string>
   </property>
  </action>
  <action name="actionSaveAs" >
   <property name="icon" >
    <iconset resource="../res.qrc" >
     <normaloff>:/Toolbar/pics/filesaveas.png</normaloff>:/Toolbar/pics/filesaveas.png</iconset>
   </property>
   <property name="text" >
    <string>Save As</string>
   </property>
   <property name="statusTip" >
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionConfigure" >
   <property name="icon" >
    <iconset resource="../res.qrc" >
     <normaloff>:/Toolbar/pics/configure.png</normaloff>:/Toolbar/pics/configure.png</iconset>
   </property>
   <property name="text" >
    <string>Configure</string>
   </property>
   <property name="statusTip" >
    <string>Show options dialog...</string>
   </property>
  </action>
  <action name="actionHelp" >
   <property name="icon" >
    <iconset resource="../res.qrc" >
     <normaloff>:/Toolbar/pics/help.png</normaloff>:/Toolbar/pics/help.png</iconset>
   </property>
   <property name="text" >
    <string>Help</string>
   </property>
   <property name="statusTip" >
    <string>Show Help</string>
   </property>
  </action>
  <action name="actionInfo" >
   <property name="icon" >
    <iconset resource="../res.qrc" >
     <normaloff>:/Toolbar/pics/info.png</normaloff>:/Toolbar/pics/info.png</iconset>
   </property>
   <property name="text" >
    <string>Info</string>
   </property>
   <property name="statusTip" >
    <string>Show Info...</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="text" >
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res.qrc" />
 </resources>
 <connections/>
</ui>
